WordpressSync/index.js

```diff
@@ -293,7 +257,42 @@ module.exports = async () => {
       
       await PrIfChanged(gitRepo, endpoint.GitHubTarget.Branch, mediaTree, `${commitTitleMedia} (${mediaTree.length} updates)`, committer, true);
     }
-    
+
+    /**
+     * Point specific WP content to static file hosting
+     * @param {*} jsonData 
+     * @param {*} WpRow 
+     * @param {*} HTML 
+     */
+    const convertWpEndpointsToStatic = (jsonData,WpRow,HTML,type) => {
+      let apiSearchString = wordPressApiUrl;
+      if (apiSearchString.charAt(apiSearchString.length - 1) === "/") {
```

Comment:
Use `endsWith()` ?  https://www.w3schools.com/Jsref/jsref_endswith.asp

Reply:
Or for this case just remove the trailing slash with replace...`.replace(/\/$/, "")`